easybuild/easyconfigs/k/KAT/KAT-2.4.2-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'ConfigureMake'
+
+name = 'KAT'
+version = '2.4.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.earlham.ac.uk/kat-tools'
+description = 'The K-mer Analysis Toolkit (KAT) contains a number of tools that analyse and compare K-mer spectra.'
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = ['https://github.com/TGAC/KAT/archive']
+sources = ['Release-%(version)s.tar.gz']
+patches = [
+    'KAT-%(version)s-use-EB-Boost.patch',
+    'KAT-%(version)s-fix-python.patch',
+]
+checksums = [
+    'd6116cefdb5ecd9ec40898dd92362afe1a76fa560abfe0f2cd29cbe0d95cb877',  # Release-2.4.2.tar.gz
+    'c8c5738a97362e0c0f88d99fd14103dca9c1928f81d967b6783aba7c2421d672',  # KAT-2.4.2-use-EB-Boost.patch
+    '566f38ced78a0d74a3740ec5913f624a2e6956011041947856164b79cfba44b3',  # KAT-2.4.2-fix-python.patch
+]
+
+builddependencies = [('Autotools', '20180311')]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('Python', '3.7.2'),
+    ('Boost', '1.70.0'),
+    ('matplotlib', '3.0.3', versionsuffix),
+]
+
+preconfigopts = "./autogen.sh && "
+
+# The test don't work during this step but work fine after the full install
+# runtest = "-C tests check V=1"
```

Comment:
@Darkless012 Let's remove these commented out lines, since they're handled via `sanity_check_paths` now?

Reply:
I think that this brings insight into why we are running make test outside of runtest phase?

Reply:
Maybe just clarify the comment below in `sanity_check_commands` to something like:

```python
# tests only work after actual installation, so running them during sanity check...
```